### DISCUSSION TITLE
QRZ

### DISCUSSION THREAD

**Discussion by lmattpatt** [magicbug/Cloudlog#1938] [Q&A] [answered]
Does Cloudlog have the ability to mark contacts confirmed via QRZ?

**Comment by phl0** [comment]
> Does Cloudlog have the ability to mark contacts confirmed via QRZ?

Nope. Currently not.

**Reply by ofadam** [reply]
If desired, it does look like the QRZ API could provide that data with the FETCH command and the STATUS:CONFIRMED option (a user would need a subscription, of course.)

https://www.qrz.com/docs/logbook/QRZLogbookAPI.html

**Reply by phl0** [reply]
But AFAIK there are no ADIF fields to handle this data. As Cloudlog does heavily rely on this we would need to find a solution here. But tbh I don't see any advantage here as those would only count for qrz.com own awards. Right?

**Reply by ofadam** [reply]
Ah, right you are on there only being QRZCOM_QSO_UPLOAD_DATE and QRZCOM_QSO_UPLOAD_STATUS fields in the ADIF spec. 

(I also forgot that I'd created a thread back in 2021 with the same discussion: https://github.com/magicbug/Cloudlog/discussions/856)

**Comment by lmattpatt** [comment]
For me personally it would offer a nice way of  having this info in one
location instead of having to go to QRZ to hunt for confirmations there,
etc.  Most logging applications have this functionality.  Thanks for the
consideration.

73 Matt
W5LL

**Comment by dg9vh** [comment]
Well, ADIF-specification supports application defined fields with the
disadvantage of being not interchangeable with other applications  - means
having no effect on this. .But for storing internal data this could be
used...

I think, this is not the way we want to go.

Kim

**Comment by magicbug** [comment, marked as answer]
Currently isn't going to be supported within Cloudlog.

This is because I don't want the headache of adding and supporting two extra database columns :)

**Reply by TheMcBain92** [reply]
This will be one feature i'll be waiting for. Hope it can be supported soon.

**Reply by magicbug** [reply]
As pointed out it wont be supported
